test-app/tests/integration/components/input-field-test.gts

```diff
@@ -0,0 +1,90 @@
+import { render } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+
+import InputField from '@crowdstrike/ember-toucan-core/components/form/input-field';
+import { setupRenderingTest } from 'test-app/tests/helpers';
+
+module('Integration | Component | Field', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(<template>
+      <InputField
+        @label="Label"
+        type="text"
+        />
+    </template>);
+
+    const label = 'label'
+    const input = 'input';
+
+    assert.dom(label).exists('Expected to have label block rendered');
+    assert.dom(label).hasText('Label', 'Expected to have label text "label"');
+
+    assert.dom(input).exists('Expected to have input tag rendered');
+    assert.dom(input).hasAttribute('type', 'text');
+  });
+  
+  test('it renders an error', async function (assert) {
+    await render(<template>
+      <InputField
+        @label="Label"
+        type="text"
+        @error="There is an error"
+        />
+    </template>);
+
+    const label = 'label'
+    const input = 'input';
+    const error = 'div.text-critical';
```

Comment:
**Question:** wondering if I should add `data-test` selectors in the https://github.com/CrowdStrike/ember-toucan-core/pull/35/files#diff-a8dfe5662c125f37bbd67d22d9112a123c0ffac52f7254a0fa217dd019452071 to get around these horribly specific class selectors?